app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,49 @@
+class AdminsController < Admins::BaseController
+  include MessageHelper
+
+  def show
+    @admin = current_admin # take this line out
```

Comment:
We need this right now. We can remove it once we set up the admin authorization.